src/browser/starter.js

```diff
@@ -607,18 +603,17 @@ V86Starter.prototype.continue_init = async function(emulator, options)
 
             if(options["bzimage_initrd_from_filesystem"])
             {
-                const { bzimage, initrd } = this.get_bzimage_initrd_from_filesystem(settings.fs9p);
+                let { bzimage, initrd } = this.get_bzimage_initrd_from_filesystem(settings.fs9p);
```

Comment:
Could you rename these variables (e.g. to bzimage_path, initrd_path) and change the let back to a const?